ft_tolower.c

```diff
@@ -0,0 +1,27 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_tolower.c                                       :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: hbalan <hbalan@student.42.fr>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/01/13 15:05:15 by hbalan            #+#    #+#             */
+/*   Updated: 2023/01/13 15:05:17 by hbalan           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <stdio.h>
+#include <ctype.h>
+
+int	ft_tolower(int c)
+{
+	if ((c >= 'A' && c <= 'Z'))
+		c += 32;
+	return (c);
+}
+
+// int main(void)
+// {
+//     printf("%d", tolower('C'));
+//     return (0);
+// }
```

Comment:
Good job! And good luck with Moulinette ;)

Reply:
Thankssss